apps/kol-amelamdim/pages/api/me.ts

```diff
@@ -1,16 +1,22 @@
 import { NextApiRequest, NextApiResponse } from 'next';
-import { API_ERRORS } from '@kol-amelamdim/api-errors';
+import { jwtVerify } from 'jose';
 
-export default function handler(req: NextApiRequest, res: NextApiResponse) {
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
   try {
     const { cookies } = req;
-
-    if (cookies.token) {
+    const { payload } = await jwtVerify(
+      cookies.token,
+      new TextEncoder().encode(process.env.ACCESS_TOKEN_SECRET)
+    );
+    if (payload) {
       return res.status(200).json({ success: true });
     }
 
     return res.status(200).json({ success: false });
   } catch (error) {
-    return res.status(404).json(API_ERRORS.GeneralError);
+    return res.status(200).json({ success: false });
```

Comment:
Why 200 for error. I think we should send 401 if i'm not wrong it's not authorise